tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -401,7 +401,12 @@ def test_fsdp_mixed_with_sync(
         '0.17.0',
         marks=pytest.mark.filterwarnings((r'ignore:MosaicMLLogger is not in the state_dict. Its '
                                           r'state will not be restored.:UserWarning')),
-    )
+    ),
+    pytest.param(
+        '0.18.1',
+        marks=pytest.mark.filterwarnings((r'ignore:MosaicMLLogger is not in the state_dict. Its '
+                                          r'state will not be restored.:UserWarning')),
+    ),
```

Comment:
I don't think this warning is needed anymore?

Reply:
Done.